derive/Cargo.toml

```diff
@@ -18,6 +18,7 @@ repository = "https://github.com/rust-fuzz/arbitrary"
 documentation = "https://docs.rs/arbitrary/"
 
 [dependencies]
+heck = "0.4"
```

Comment:
hmm, `heck` is a trustable dep, but how much do we care about adding deps @fitzgen?

we could simply use `.toUppercase()` here, it doesn't have to look good since it's generated code.

Reply:
Good point; I'd be okay with that change. I'm not sure how much the pathological case of deriving on, say, `CustomThing` and `Customthing` in the same module is an actual problem, but `heck` would cover that case while uppercasing wouldn't.

Reply:
Yeah, I'm aware. Unsure how much I care about it. We could also just `#[allow]` the lint

Reply:
That's probably the easiest/safest option actually; should we just do that?

Reply:
yeah!

Reply:
Done